.gitignore

```diff
@@ -44,3 +44,4 @@ package-lock.json
 yarn.*
 yarn-error.*
 .nyc_output
+yarn.lock
```

Comment:
@abayo-luc `yarn.*`  includes `yarn.lock` no need to add it again

Reply:
That one is not working properly.